R/utils.R

```diff
@@ -61,33 +61,27 @@ transform_params <- function(params){
   params_string <- try(eval(parse(text = paste('alist(', quote_label(params), ')'))),
                        silent = TRUE)
 
-  if(inherits(params_string, "try-error")){
+  if (inherits(params_string, "try-error")){
```

Comment:
```suggestion
  if (inherits(params_string, "try-error")) {
```

Even more whitespace :grin:

Reply:
good catch!